cisco_aci/README.md

```diff
@@ -104,6 +104,30 @@ Specific supported vendor profiles for this integration can be found on the [net
 
 See [metadata.csv][7] for a list of metrics provided by this integration.
 
+### Cisco ACI faults
+
+<div class="alert alert-info">Billing impact callout</div>
+
+Cisco ACI faults are collected by the Agent and stored in Datadog as logs. They can be viewed from the [Cisco ACI dashboard][15], or by filtering for `source:cisco-aci` in the [Log Explorer][16].
+
+There are two types of Cisco ACI faults: `fault:Inst` and `fault:Delegate`. See [fault objects and records][12] for more information.
+ 
+The Cisco ACI config `.yaml` file can be configured to collect one of the following fault types:
+
+| Fault Type          | Description                                                                 |
+|---------------------|-----------------------------------------------------------------------------|
+| [send_faultinst_faults][13]     | Set to `true` to enable collection of Cisco ACI `faultInst` faults as logs. |
+| [send_faultdelegate_faults][14] | Set to `true` to enable collection of Cisco ACI `faultDelegate` faults as logs. |
+
+If you have enabled either `send_faultinst_faults` or `send_faultdelegate_faults`, add the following configuration to the logs section of your `.yaml` file:
+
+```yaml
+logs:
+  - type: integration
+    source: cisco-aci
+    service: cisco-aci
+```
+
```

Comment:
In addition to this config the agent also needs to have logs enabled.  I added a comment to the Jira card.